common/testData/personNameTest/mt.txt

```diff
@@ -0,0 +1,239 @@
+# Test data for unit conversions
+#  Copyright © 1991-2023 Unicode, Inc.
+#  For terms of use, see http://www.unicode.org/copyright.html
+#  SPDX-License-Identifier: Unicode-DFS-2016
+#  CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+#
+# CLDR person name formatting test data for: mt
```

Comment:
This looks like a whole new data file.  We didn't have person-name data for this locale before?

Reply:
The test generates data if there are two fields with explicit values in the file: 
//ldml/personNames/nameOrderLocales[@order=\"givenFirst\"]
//ldml/personNames/nameOrderLocales[@order=\"surnameFirst\"]

So if people added values in the SurveyTool we generate the test file for the locale in that release.

Reply:
I see.  Thanks for the explanation.